jaeger_client/config.py

```diff
@@ -219,9 +219,11 @@ def max_traceback_length(self):
 
     @property
     def sampler(self):
-        sampler_config = self.config.get('sampler', {})
-        sampler_type = sampler_config.get('type', None)
-        sampler_param = sampler_config.get('param', None)
+        sampler_param = self.config.get('sampler', {})
```

Comment:
this is overloading `sampler_param` variable. I would keep it as `sampler_config`

Reply:
I choose to change from `sampler_config` to `sampler_param` as now it can be both a sampler or a config so I choose a more generic name  but if you want I can come back to `sampler_config`

Reply:
I'd be fine with `sampler_config` or `sampler` (prefer config as it's more common use case). But `param` is confusing since it's the inner field.

Reply:
you are right . I come back to sampler_config .